### PR TITLE
Design enhancements

### PR DESCRIPTION
**Pull request by jancborchardt**
Just 2 small enhancements to the detail view header:
- make avatar round, like all others
- add placeholder »Name« to name input so it’s more visible when empty

### CODE REVIEW

File: templates/contactDetails.html [hunk 1]
```diff
@@ -6,9 +6,9 @@
 </div>
 <div ng-if="ctrl.contact!==undefined">
 <header class="contactdetails__header" ng-style="{'background-color': (ctrl.contact.uid() | contactColor)}">
-	<img class="contactdetails__logo" data-ng-src="data:image/png;base64,{{ctrl.photo}}" />
+	<img class="contactdetails__logo avatar" data-ng-src="data:image/png;base64,{{ctrl.photo}}" />
 	<h2>
-		<input type="text" id="details-email" class="contactdetails__name" name="email" ng-model="ctrl.contact.fullName" ng-model-options="{ getterSetter: true, debounce: 500 }" ng-change="ctrl.updateContact()" value="" />
+		<input type="text" id="details-email" class="contactdetails__name" placeholder="Name" name="email" ng-model="ctrl.contact.fullName" ng-model-options="{ getterSetter: true, debounce: 500 }" ng-change="ctrl.updateContact()" value="" />
```
Comment (DeepDiver1975) [inline templates/contactDetails.html]: We need to translate that

**Comment by DeepDiver1975** [comment]
no so cool if the photo is not square

bildschirmfoto von 2016-02-26 07-36-46

**Comment by jancborchardt** [comment]
The photo should always be cut to display square. Or at least we should force it square via CSS. Smth for a different PR though.

**Comment by DeepDiver1975** [comment]
> Or at least we should force it square via CSS

we should do this - pr welcome

**Comment by Henni** [comment]
looks good otherwise :+1:

**Pull request merged**